src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,45 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
+    {
+        $taxTotal = 0;
+
+        foreach ($this->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT) as $taxAdjustment) {
+            $taxTotal += $taxAdjustment->getAmount();
+        }
+
+        foreach ($this->units as $unit) {
+            $taxTotal += $unit->getTaxTotal();
+        }
+
+        return $taxTotal;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDiscountedUnitPrice()
+    {
+        $orderItemDiscounts = 0;
+        foreach ($this->units as $unit) {
+            $orderItemDiscounts += $unit->getAdjustmentsTotal(AdjustmentInterface::ORDER_ITEM_PROMOTION_ADJUSTMENT);
```

Comment:
Right now there is no action that would give you a fixed discount on `OrderItem`, all actions are per unit. There is no rounding at all because the discount is always equal on all units.